staging/src/k8s.io/kubectl/pkg/cmd/completion/completion.go

```diff
@@ -158,7 +158,6 @@ func runCompletionZsh(out io.Writer, boilerPlate string, kubectl *cobra.Command)
 __kubectl_bash_source() {
 	alias shopt=':'
 	alias _expand=_bash_expand
-	alias _complete=_bash_comp
```

Comment:
per https://github.com/kubernetes/kubectl/issues/125#issuecomment-392754413, _bash_expand is also undefined. I don't have the context to know what should be done here, will defer to @kubernetes/sig-cli-maintainers

Reply:
confirmed and tested locally.  Removed _expand alias.